### DISCUSSION TITLE
Unable to get a live stream for Doorbell Dual (T8203)

### DISCUSSION THREAD

**Discussion by TheJakeMoore** [fuatakgun/eufy_security#640] [Q&A]
Hey everyone,

I've got all of my Eufy devices up and running in HA, except for my doorbell. The device and its entities are all discovered in HA, but the camera.doorbell entity isn't able to show a live stream. The doorbell dual does not support RTSP.

What I'm currently doing is just a picture entity card that shows camera.doorbell.
```
show_state: false
show_name: false
camera_view: live
type: picture-entity
entity: camera.doorbell
image: https://demo.home-assistant.io/stub_config/bedroom.png
camera_image: camera.doorbell
```

Two things are happening:

1. The camera stream only works in the HA native app. It doesn't work in a browser. In a browser, it shows no video and flashes a warning that reads "Error with media stream controls"
2. When viewing the camera stream on the native app, it only shows a still frame. When tapping on the card, it pulls up about a 20sec clip from some time in the past. I haven't determined what is dictating the time it chooses.

The add-on log is producing this error:
```
2023-01-04 02:42:10.593  ERROR Message error 
 LivestreamNotRunningError  Livestream for device T8203P11224304C5 could not be stopped, because it is not running
error stack:
• message_handler.ts:240 handle
    src/lib/device/message_handler.ts:240:31
• task_queues:96 processTicksAndRejections
    node:internal/process/task_queues:96:5
• server.ts:125 receiveMessage
    src/lib/server.ts:125:21
```

And I'm seeing the following in my system log:
```
2023-01-04 07:53:58.681 ERROR (Thread-10 (handle_queue_threaded)) [custom_components.eufy_security] eufy_security Doorbell - stop_ffmpeg exception: 'NoneType' object has no attribute 'kill' - traceback: Traceback (most recent call last):
File "/config/custom_components/eufy_security/camera.py", line 296, in stop_ffmpeg
self.ffmpeg.kill()
File "/usr/local/lib/python3.10/site-packages/haffmpeg/core.py", line 170, in kill
self._proc.kill()
AttributeError: 'NoneType' object has no attribute 'kill'
...
2023-01-04 07:54:01.258 ERROR (Thread-10 (handle_queue_threaded)) [custom_components.eufy_security] eufy_security Doorbell - handle_queue_threaded - Unable to send payload - [Errno 32] Broken pipe
```

**Comment by kauthmbt** [comment]
Hi @TheJakeMoore,

Are you already using the latest version?
I am using T8210 which also doesn't support RTSP and I got it running exactly like mentioned in chapter 4: https://github.com/fuatakgun/eufy_security#4-setting-up-your-dashboard-for-camera
